Makefile

```
@@ -26,7 +26,10 @@ doc/content/shortcodes.md: $(wildcard layouts/shortcodes/*.html)
 # Serve for development purposes.
 serve-dev: doc-serve
 doc-serve: doc/content/shortcodes.md
-	(cd doc && hugo --printI18nWarnings serve --themesDir="../.." --disableFastRender --poll 1000ms)
+	(cd doc; \
+	hugo; \
+	yes | npx pagefind --site public; \
```

```suggestion
	yes | npx pagefind --site public; \
```

Maybe flipping yes and npx would be slightly more intuitive. Your call.

---

Agreed - I piped the `yes` command because I didn't initially see that npx supports a `--yes` flag; but it turned up buried on closer reading.